golang.go

```diff
@@ -0,0 +1,19 @@
+// package docummentation
+package main
+
+import "context"
+
+// here is a line coment
+
+// FileGetter is used to retriv the code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
+	GetFiles(context.Context, *FilesRequest) (FileScanner, error)
+}
+
+func main() {
+	/*
```

Comment:
Typo inside comment in word `coment`. Possible corrections: comment (64), cement (47), moment (47),